src/components/backend-ai-project-switcher.ts

```diff
@@ -0,0 +1,127 @@
+/**
+ @license
+ Copyright (c) 2015-2022 Lablup Inc. All rights reserved.
+ */
+
+
+import {customElement, property, query} from 'lit/decorators.js';
+import {LitElement, html, CSSResultGroup} from 'lit';
+import {translate as _t} from 'lit-translate';
+import '@material/mwc-select';
+
+import {BackendAIWebUIStyles} from './backend-ai-webui-styles';
+import {
+  IronFlex,
+  IronFlexAlignment,
+  IronFlexFactors,
+  IronPositioning
+} from '../plastics/layout/iron-flex-layout-classes';
+import {get as _text} from 'lit-translate/util';
+
+/**
+ Backend AI Project Switcher
+
+ Example:
+
+ <backend-ai-project-switcher></backend-ai-project-switcher>
+
+ @group Backend.AI Web UI
+ @element backend-ai-project-switcher
+ */
+@customElement('backend-ai-project-switcher')
+export default class BackendAIProjectSwitcher extends LitElement {
+  @property({type: Array}) groups = [];
+  @property({type: String}) current_group = '';
```

Comment:
Oops, I found another one that doesn't fit the naming convention.

Reply:
Updated.